cmd/stack_create.go

```diff
@@ -111,23 +110,43 @@ The stack name must start with a lowercase letter, and can contain only lowercas
 				Info.logf("Dry Run -Skipping moving out of stack: %s from %s", projectType, filepath.Join(stack, "stacks-master", config.copy))
 
 			} else {
-				err = os.Rename(filepath.Join(stack, "stacks-master", config.copy), projectType)
+				ExistingStackFolderExist, err = Exists(projectType)
```

Comment:
Is this `projectType` variable used for anything other than naming this temporary directory? 
A couple suggestions:
1. Give `projectType` a better name. If it is supposed to refer to the stack, call it `stackID`. If it is just used for a temporary directory name, call it something like `stackTempDir`
2. Can you always set the directory name with the timestamp? That way you don't have to use `ExistingStackFolderExist` and have multiple code branches everywhere.
Maybe even start it with a dot (to make it hidden) and include `temp` in the name. That way if it doesn't get cleaned up the it will be clear to the user that it is temporary.
like this `.temp-nodejs-express-20060102150405`

Reply:
I have made changes. Thanks for the useful comments.